scania-digital-design-system/sdds
# investigation(card/textfield): mode/variant exploration

## `🚧 DO NOT MERGE 🚧`

**Describe pull-request**  
Investigation to see how we could work with variant and dark/light-mode. 

**Solving issue**  
Fixes: [AB#2462](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2462)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check the components
3. Try them out on dark/light mode and both variants.

**Screenshots**  
-

**Additional context**  
-
